django/core/cache/backends/base.py

```diff
@@ -78,6 +82,10 @@ def __init__(self, params):
         self.version = params.get('VERSION', 1)
         self.key_func = get_key_func(params.get('KEY_FUNCTION'))
 
+    @cached_property
+    def _sentinel(self):
+        return object() if self.supports_get_with_default else None
```

Comment:
I think we can move `supports_get_with_default` to test classes, revert changes in `BaseMemcachedCache` and `MemcachedCache`, and change `_sentinel` to a class property:
```python
class BaseCache:
    _missing_key = object()
```
What do you think? this should simplify removing `MemcachedCache` when deprecation ends.

Reply:
Yes. I thought about it, but then didn't because I didn't think to move the `supports_get_with_default` flag into the tests. You suggestion will definitely slim it down.